discord/emoji.py

```diff
@@ -281,7 +281,7 @@ def is_usable(self):
 
         await self._state.http.delete_custom_emoji(self.guild.id, self.id, reason=reason)
 
-    async def edit(self, *, name, roles=None, reason=None):
+    async def edit(self, *, reason=None, **fields):
```

Comment:
```suggestion
    async def edit(self, *, name=None, roles=None, reason=None):
```

I don't get the pointless `**fields` transform since it's complicating the body.